TTInAppPurchases/View Controllers/SubscriptionFlow/WeeklyMonthlyAndAnnualViewController.swift

```diff
@@ -0,0 +1,318 @@
+    private func _drawShape() {
+        let path = UIBezierPath()
+        path.move(to: CGPoint(x: 0, y: 0))
+        path.addLine(to: CGPoint(x: view.bounds.size.width * 0.65, y: 0))
+        
+        let QPoint1 = CGPoint(x: view.bounds.size.width * 0.40, y: view.bounds.size.height * 0.15)
+        let QPoint2 = CGPoint(x: view.bounds.size.width * 0.30, y: view.bounds.size.height * 0.02)
+        
+        path.addCurve(to: CGPoint(x: 0, y: view.bounds.size.width * 0.30),
+                      controlPoint1: QPoint1,
+                      controlPoint2: QPoint2)
+        path.close()
+        
+        let layer1 = CAShapeLayer()
+        layer1.fillColor = UIColor.primaryColor.cgColor
+        layer1.path = path.cgPath
+        contentView.layer.insertSublayer(layer1, at: 1)
+        
+        let layer2 = CAShapeLayer()
+        layer2.fillColor = UIColor.primaryColor.withAlphaComponent(0.4).cgColor
+        layer2.path = path.cgPath
+        layer2.transform = CATransform3DMakeScale(1.13, 1.10, 0)
+        contentView.layer.insertSublayer(layer2, below: layer1)
+    }
+    
+    private func _configureUI() {
+        if bounds.height >= 896 {
+            // 12 PRO MAX, 11 PRO MAX, XS MAX, XR , 11
+            featureLabelTextStyle = bounds.height >= 926 ? .title3 : .headline
+            // 12 PRO MAX
+            stackViewHeightConstraint.constant = bounds.height >= 926 ? 170 : 160
+        } else {
+            // all the rest
+            featureLabelTextStyle = .body
+        }
+    }
+    
+    // TODO: Add localized text for dummy header and description labels
+    private func _configureHeaderLabels() {
+        primaryHeaderLabel.configure(with: UIFont.font(.sofiaProBold, style: .title2))
+        primaryHeaderLabel.text = "Upgrade To Premium"
+        
+        bottomHeaderLabel.configure(with: UIFont.font(.sofiaProSemibold, style: .subheadline))
+        bottomHeaderLabel.text = "Download, Try and Test the App"
+    }
+    
+    private func _configureDescriptionLabels() {
+        topDescriptionLabel.configure(with: UIFont.font(.sofiaProLight, style: .caption2))
+        topDescriptionLabel.text = "EZTape Call Recorder is the simplest and most seamless recording app on the app store. This business app allows you to record your incoming and outgoing phone calls."
+        
+        bottomDescriptionLabel.configure(with: UIFont.font(.sofiaProRegular, style: .caption1))
+        bottomDescriptionLabel.text = "Call recordings like never before on your iOS device. Call, Record*, Store and Share the call with your teammates. *We are supporting the online and outdoor call recording where you don’t have anything to write the important stuff. We are not breaching anyone’s privacy policy."
+    }
+    
+    private func _configureFeatureLabel() {
+        feature1Label.configure(with: UIFont.font(.sofiaProMedium, style: .caption2))
+        feature1Label.text = "Automatic call recordings or tap call merging"
```

Comment:
Please add all the strings to the .strings file. Reuse similar strings from the file.